packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -33,6 +33,15 @@ const { trackBuildError, decorateEvent } = require(`gatsby-telemetry`)
 import errorParser from "./api-runner-error-parser"
 const { loadNodeContent } = require(`../db/nodes`)
 
+if (!process.env.BLUEBIRD_DEBUG && !process.env.BLUEBIRD_LONG_STACK_TRACES) {
+  // Unless specified - disable longStackTraces
+  // as this have severe perf penalty ( http://bluebirdjs.com/docs/api/promise.longstacktraces.html )
+  // This is main for `gatsby develop` due to NODE_ENV being set to development
```

Comment:
s/main/mainly/

Reply:
Addressed in https://github.com/gatsbyjs/gatsby/pull/28957/commits/92c5ced34ef041aea92267c8bf035a4e805f0dee